src/components/map/KmlService.js

```diff
@@ -153,7 +153,7 @@ goog.require('ga_urlutils_service');
           // OL applies a default scale multiplier to 0.5
           // https://github.com/openlayers/ol3/blob/master/src/ol/format/kml.js#L622
           // https://github.com/openlayers/ol3/pull/5745
-          if (image && image.getScale()) {
+          if (image && image.getScale() && image.getScale() != 1) {
```

Comment:
Yes ok, thank you. This scale stuff really makes no sense at the moment but it's the best fix we could do.